### DISCUSSION TITLE
Tag hierarchy problem

### DISCUSSION THREAD

**Discussion by jkilopu** [marcusolsson/obsidian-projects#549] [Help] [answered]
I have a note tagged with #calculus/derivative. However, when I created a tag project with #calculus, the note didn't appear.
I'm not sure if I'm missing something or if this behavior is intentional, so I'm posting here.
Thank you so much!

**Comment by marcusolsson** [comment, marked as answer]
The Tag data source doesn't take tag hierarchy into account.

Unfortunately, introducing it now could break people's projects I'm afraid. One solution would be to make it configurable, with tag hierarchies disabled by default.

**Reply by Xelbayria** [reply]
I'm curious, how would one make it configurable?

**Reply by Acylation** [reply]
Hi! In the latest 1.16.5 release, you can configure your tag based project to enable tag hierarchy.

![image](https://github.com/marcusolsson/obsidian-projects/assets/73122375/a21c144e-de6a-4dd5-b9a6-36a4c23afbb0)
